src/20_Components/amp-mustache.html

```diff
@@ -0,0 +1,111 @@
+<!--
+  ## Introduction
+
+  <a href="https://www.ampproject.org/docs/reference/components/amp-mustache">amp-mustache</a> templates are a simple, structured templating system.
```

Comment:
nit: Use markdown syntax instead of html: `[amp-mustache](https://www.ampproject.org/docs/reference/components/amp-mustache)`.